mathics/builtin/box/graphics3d.py

```diff
@@ -776,12 +776,13 @@ def _apply_boxscaling(self, boxscale):
         pass
 
 
-class Cylinder3DBox(InstanceableBuiltin):
+class Cylinder3DBox(Graphics3DBox):
```

Comment:
`InstanceableBuiltin` is faster.

Reply:
> `InstanceableBuiltin` is faster.

But it is wrong. Cylinders are not unique, like the Number 5 is. And we need a graphics object to store whether we have imported `solds`.